tests/test_metadata.py

```diff
@@ -684,6 +684,12 @@ def test_optional_defaults_to_none(self, field_name):
                 "(LicenseRef-Special-License OR licenseref-OtherLicense) OR unlicense",
                 "(LicenseRef-Special-License OR LicenseRef-OtherLicense) OR Unlicense",
             ),
+            # we don't canonicalize redundant parens, instead leaving them as-is
+            # in the license expression.
+            ("(MIT)", "(MIT)"),
+            ("((MIT))", "((MIT))"),
+            ("(( MIT ))", "((MIT))"),
+            ("((MIT AND (MIT)))", "((MIT AND (MIT)))"),
```

Comment:
we could make it so redundant parens are canonicalized, but we'd have to modify the parser to be a proper parser instead of deferring to python's `compile`.